mne/io/cnt/cnt.py

```diff
@@ -221,31 +293,39 @@ def _get_cnt_info(input_fname, eog, ecg, emg, misc, data_format, date_format):
             cal = np.fromfile(fid, dtype='f4', count=1)
             cals.append(cal * sensitivity * 1e-6 / 204.8)
 
-        if event_offset > data_offset:
-            fid.seek(event_offset)
-            event_type = np.fromfile(fid, dtype='<i1', count=1)[0]
-            event_size = np.fromfile(fid, dtype='<i4', count=1)[0]
-            if event_type == 1:
-                event_bytes = 8
-            elif event_type in (2, 3):
-                event_bytes = 19
+        if stim_channel_toggle:
+            if event_offset > data_offset:
+                fid.seek(event_offset)
+                event_type = np.fromfile(fid, dtype='<i1', count=1)[0]
+                event_size = np.fromfile(fid, dtype='<i4', count=1)[0]
+                teeg_offset = np.fromfile(fid, dtype='<i4', count=1)[0]
+                assert teeg_offset == 0  # documentation say this should be 0
+                if event_type == 1:
+                    event_bytes = 8
+                elif event_type in (2, 3):
+                    event_bytes = 19
+                else:
+                    raise IOError('Unexpected event size.')
+
+                # XXX long NumEvents is available, why are not we using it?
+                n_events = event_size // event_bytes
             else:
-                raise IOError('Unexpected event size.')
-            n_events = event_size // event_bytes
-        else:
-            n_events = 0
-
-        stim_channel = np.zeros(n_samples)  # Construct stim channel
-        for i in range(n_events):
-            fid.seek(event_offset + 9 + i * event_bytes)
-            event_id = np.fromfile(fid, dtype='u2', count=1)[0]
-            fid.seek(event_offset + 9 + i * event_bytes + 4)
-            offset = np.fromfile(fid, dtype='<i4', count=1)[0]
-            if event_type == 3:
-                offset *= n_bytes * n_channels
-            event_time = offset - 900 - 75 * n_channels
-            event_time //= n_channels * n_bytes
-            stim_channel[event_time - 1] = event_id
+                n_events = 0
+
+            stim_channel = np.zeros(n_samples)  # Construct stim channel
+            for i in range(n_events):
+                fid.seek(event_offset + 9 + i * event_bytes)
+                event_id = np.fromfile(fid, dtype='u2', count=1)[0]
+                fid.seek(event_offset + 9 + i * event_bytes + 4)
+                offset = np.fromfile(fid, dtype='<i4', count=1)[0]
+                if event_type == 3:
+                    offset *= n_bytes * n_channels
+                event_time = offset - 900 - 75 * n_channels
+                event_time //= n_channels * n_bytes
+                stim_channel[event_time - 1] = event_id
+
+        else:  # when stim_channel_toggle is False
+            pass
```

Comment:
I don't see much value in `else: pass` because it's implied by just not having it there; and in this case the comment does not add anything because it's just the conditional checked above